gapis/api/transform2/transform_test.go

```diff
@@ -0,0 +1,126 @@
+// Copyright (C) 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package transform2 contains the the elements to be able to transform
+// commands which consist of interfaces for individual transform operations
+// and a transform chain to run all of them.
```

Comment:
This is redundant with the comment in `trasnform.go`, I think it's a left-over from copy-pasting, you can probably remove it.